files/class-api-client.php

```diff
@@ -145,8 +145,9 @@ public function upload_file( $local_path, $upload_path ) {
 		// save to cache
 		$this->cache->copy_to_cache( $response_data->filename, $local_path );
```

Comment:
We should also change this here, as we are also copying the file to the wrong cache key. Not as critical as the file stats cache, but at least that the cached file can be used.

Reply:
Good thinking! Definitely makes sense to do here.

I'm wondering thought if we should just account for the leading slash in the Cache class because the same problem is bound to come up again.